"make setup" don't work

Hi,

**Describe the bug**
`mogel@lucretia:~/Projekte/VCU/opengrep
 (main) $ make setup
./scripts/check-bash-version
Error: Bash >= 5.0 is required.
If you're on a Mac, run:

  brew install bash

make: *** [Makefile:496: setup] Fehler 1
`

**To Reproduce**
in an empty directory:

`git clone ...
git submodule update ...
make setup
`

**Expected behavior**
Setup working

**Environment**
`mogel@lucretia:~/Projekte/VCU/opengrep
 (main) $ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=24.04
DISTRIB_CODENAME=noble
DISTRIB_DESCRIPTION="Ubuntu 24.04.1 LTS"
`

`mogel@lucretia:~/Projekte/VCU/opengrep
 (main) $ echo $SHELL && $SHELL --version
/bin/bash
GNU bash, Version 5.2.21(1)-release (x86_64-pc-linux-gnu)
Copyright (C) 2022 Free Software Foundation, Inc.
Lizenz GPLv3+: GNU GPL Version 3 oder jünger <http://gnu.org/licenses/gpl.html>

Dies ist freie Software.  Sie darf verändert und verteilt werden.
Es wird keine Garantie gewährt, soweit das Gesetz es zulässt.
`

~ mogel

--- a/scripts/check-bash-version
+++ b/scripts/check-bash-version
@@ -7,7 +7,7 @@
 #
 set -eu -o pipefail
 
-if ! { bash --version | grep -q 'version [56789]'; }; then
+if ! { bash --version | grep -iq 'version [56789]'; }; then
   cat >&2 <<EOF
 Error: Bash >= 5.0 is required.
 If you're on a Mac, run:

